src/lowlevel.ts

```diff
@@ -106,7 +150,7 @@ export function extractColumn<T = unknown, K extends keyof T = keyof T>(
   const array = Array.isArray(values) ? values : [values];
   return array.map((row: T) => {
     if (row[colName] === undefined) {
-      throw new Error(`column not found: ${colName.toString()}`);
+      throw new Error(`no such column: ${colName.toString()}`);
```

Comment:
This kind of error can come from the Validator or the client.  This changes the client phrase to match the Validator's phrase.

Reply:
nice, this will help with error-related docs, too...more unified docs!